src/development/tools/devtools/memory.md

```diff
@@ -435,86 +464,93 @@ dropdowns that control how memory data is displayed.
 <dt markdown="1">**Android Memory**</dt>
 <dd>Displays or hides the Android Memory Chart.</dd>
 <dt markdown="1">**GC**</dt>
-<dd>Initiates a garbage collection - compaction of the heap.</dd>
+<dd>Initiates a garbage collection&mdash;a compaction of the heap.</dd>
 <dt markdown="1">**Export**</dt>
-<dd>Saves collected data for Event Timeline, Memory Overview Chart
-    and Android Overview Chart. Files saved are displayed under the Source
-    dropdown. Selecting a file loads the offline data.</dd>
+<dd>Saves collected data for the Event Timeline, Memory Overview Chart,
+    and Android Overview Chart. Files saved are displayed under the
+    **Source** dropdown. Selecting a file loads the offline data.</dd>
 </dl>
 
 ## Memory actions
 
-Below the memory charts (Event Timeline, Memory Overview and Android Overview
-charts) are interactive actions used to collect and analyze information about
-memory usage while using the application DevTools is connected to there are two tabs:
+Below the memory charts (Event Timeline, Memory Overview, and Android Overview
+charts), are interactive actions used to collect and analyze information about
+memory usage while using the application that DevTools is connected.
+Two tabs are displayed:
 
 ![Two Tabs Memory Actions]({{site.url}}/assets/images/docs/tools/devtools/memory_two_tabs.png)
 
 ### Analysis tab
 
-The Analysis tab collects memory snapshots both user initiated and
-auto-collected by DevTools, when DevTools detects memory spikes.
-Each snapshot is analyzed and an analysis is created too.
+The **Analysis** tab collects memory snapshots, both user initiated and
+auto-collected, by DevTools (when DevTools detects memory spikes).
+Each snapshot is analyzed.
 
 ### Analysis actions
 
 The actions available for Analysis are:
 
 ![Screenshot of a memory actions]({{site.url}}/assets/images/docs/tools/devtools/memory_analysis_actions.png)
 
-**Snapshot** Clicking the Snapshot button makes a request to the
-Dart VM to collect the current state of memory.  The
-memory objects can be sorted by attributes such as class
-name, size, allocated instances (see [Snapshot classes](#snapshots)).
+**Snapshot**
+: Clicking the **Snapshot** button makes a request to the
+  Dart VM to collect the current state of memory. The
+  memory objects can be sorted by attributes, such as class
+  name, size, and allocated instances.
+  For more information, see [Snapshot classes](#snapshots)).
 
-**Treemap** If the Treemap switch is on the snapshot displays currently
-active memory objects, the last snapshot, memory in a high-level
-view as a tree map. (TBD details).
+**Treemap**
+: If the **Treemap** switch is on, the snapshot displays currently
+  active memory objects, the last snapshot, and memory in a high-level
+  view as a tree map. (TBD details).
```

Comment:
can we remove TBD details?

Reply:
Done (Thx, because I wasn't sure about that.)